apps/vmq_enhanced_auth/src/vmq_enhanced_auth.erl

```diff
@@ -558,36 +613,69 @@ simple_acl(_) ->
         <<"# ACL for user 'test'\n">>,
         <<"user test\n">>,
         <<"topic x/y/z/#\n">>,
+        <<"# ACL with label\n">>,
+        <<"topic read x/y/z label simple_read\n">>,
+        <<"topic write x/y/z label simple_write\n">>,
+        <<"# ACL for user 'test_user'\n">>,
+        <<"user test_user\n">>,
+        <<"topic a/b/c/# label simple_read_write\n">>,
         <<"# some patterns\n">>,
-        <<"pattern read %m/%u/%c\n">>,
```

Comment:
Why have you removed these test cases? Are you changes not supporting them?

Reply:
Added back the cases.